docs/faq/integration.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+sidebar_label: Integration
+---
+
+# Venom Blockchain FAQ: Integration with Venom Blockchain
+
+<details>
+<summary>
+How to connect Venom Wallet to my dApp?
+</summary>
+
+The easiest way to connect your dApp to Venom Wallet is to use [Venom Connect](https://www.npmjs.com/package/venom-connect). It is a library that allows you to connect to Venom Wallet (both mobile and browser extension) and interact with it. This library provides you with a handy interface for building connect popup for our venom wallet and then gives us an interface for working with the venom network.
+
+Check out [this](../build/development-guides/how-to-create-your-own-fungible-tip-3-token/venom-in-action/extend-our-tokensale-with-frontend.md#connecting-venom-wallet-to-your-app) paragraph of the frontend guide, that explains how to use venom-connect in your project. Moreover, you can check the final source code of this guide [here](https://github.com/venom-blockchain/guides/tree/master/tokensale-frontend).
+
+You can read about all configuration options in venom-connect official [repository](https://github.com/web3sp/venom-connect). Also, it has an [example](https://github.com/web3sp/venom-connect/tree/main/examples/react).
+</details>
+
+<details>
+<summary>
+What if I need very specific logic for connecting the Wallet to my dApp?
+</summary>
+
+So, in this case, you can use the library that venom-connect has been built on [inpage-provider](https://github.com/broxus/everscale-inpage-provider) and [standalone-client](https://github.com/broxus/everscale-standalone-client) - basic libraries for interaction with the venom network, so you can build your system for wallet connection. Check the documentation for these libraries in its repositories for more information.
+
+</details>
+
+<details>
+<summary>
+I need to call a smart contract method before the user connects the wallet. How can I do it?
+</summary>
+
+You should use [standalone-client](https://github.com/broxus/everscale-standalone-client) as a fallback for [inpage-provider](https://github.com/broxus/everscale-inpage-provider). It allows you to call smart contract's get methods without sending any transactions. Library venom-connect also gives you access to the standalone interface. You can use [getStandalone](https://github.com/web3sp/venom-connect#getstandalone) method to achieve this. You can check [this](../build/development-guides/how-to-create-your-own-non-fungible-tip-4-token/venom-in-action/frontend-for-nft-auction.md) guide when we use the standalone-client from `getStandalone` method to get the current auction information.
```

Comment:
You should use the [standalone-client](https://github.com/broxus/everscale-standalone-client) as a fallback for [inpage-provider](https://github.com/broxus/everscale-inpage-provider). It allows you to call smart contract's get methods without sending any transactions. The venom-connect library also gives you access to the standalone interface. You can use the [getStandalone](https://github.com/web3sp/venom-connect#getstandalone) method to achieve this. You can check this guide, where we use the standalone-client from the getStandalone method to get the current auction information.